test/test_pebble.py

```diff
@@ -344,7 +347,7 @@ def test_change_init(self):
         self.assertEqual(change.data, {})
 
     def test_change_from_dict(self):
-        d = {
+        d: 'pebble._ChangeDict' = {
```

Comment:
I guess the need for the quotes will go away once we remove the `if TYPE_CHECKING` thingy. (No need to fix that here.)